telescope/query.py

```diff
@@ -257,9 +267,14 @@ def _add_server_ips_conditional(self, server_ips, is_web100):
     else:
       local_ip_fieldname = 'connection_spec.server_ip'
 
-    self._conditional_dict['server_ip'] = []
+    self._conditional_dict['server_ips'] = []
     for server_ip in unique_server_ips:
       new_statement = '{local_ip_fieldname} = \'{server_ip}\''.format(
           local_ip_fieldname=local_ip_fieldname, server_ip=server_ip)
-      self._conditional_dict['server_ip'].append(new_statement)
+      self._conditional_dict['server_ips'].append(new_statement)
 
+  def _add_client_country_conditional(self, client_country):
+    self._conditional_dict['client_country'] = ("{fieldname} = " +
+        "'{client_country}'").format(fieldname =
+        "connection_spec.client_geolocation.country_code",
```

Comment:
I think this should just be part of the format string since it's a static string and splitting it out this way adds complexity.